### PR TITLE
refactor: Remove registrar remainders and rename LAUNCH_REGISTRAR

### PR DESCRIPTION
**Pull request by leblowl**
Rename LAUNCH_REGISTRAR to SEND_COMMUNITY_CA_DATA since it was only being used for that purpose.

### Pull Request Checklist

- [ ] I have linked this PR to related GitHub issue.
- [x] I have updated the CHANGELOG.md file with relevant changes (the file is located at the root of monorepo).

### (Optional) Mobile checklist

Please ensure you completed the following checks if you did any changes to the mobile package:

- [ ] I have run e2e tests for mobile
- [ ] I have updated base screenshots for visual regression tests

### CODE REVIEW

File: packages/backend/src/nest/connections-manager/connections-manager.service.ts [hunk 1]
```diff
@@ -131,13 +130,12 @@ export class ConnectionsManagerService extends EventEmitter implements OnModuleI
...
     console.log('init')
     this.communityState = ServiceState.DEFAULT
-    this.registrarState = ServiceState.DEFAULT
     await this.generatePorts()
     if (!this.configOptions.httpTunnelPort) {
       this.configOptions.httpTunnelPort = await getPort()
     }
 
-    this.attachsocketServiceListeners()
+    this.attachSocketServiceListeners()
```
Comment (leblowl) [inline packages/backend/src/nest/connections-manager/connections-manager.service.ts:138]: Fix typo

File: packages/backend/src/nest/connections-manager/connections-manager.service.ts [hunk 2]
```diff
@@ -492,14 +486,11 @@ export class ConnectionsManagerService extends EventEmitter implements OnModuleI
...
     })
-    this.socketService.on(SocketActionTypes.LAUNCH_REGISTRAR, async (args: LaunchRegistrarPayload) => {
-      // Event left for setting permsData purposes
-      this.logger(`socketService - ${SocketActionTypes.LAUNCH_REGISTRAR}`)
-      this.registrationService.permsData = {
-        certificate: args.rootCertString,
-        privKey: args.rootKeyString,
-      }
+    this.socketService.on(SocketActionTypes.LEAVE_COMMUNITY, async () => {
+      await this.leaveCommunity()
```
Comment (leblowl) [inline packages/backend/src/nest/connections-manager/connections-manager.service.ts:490]: Just re-organizing

File: packages/backend/src/nest/connections-manager/connections-manager.service.ts [hunk 3]
```diff
@@ -511,11 +502,35 @@ export class ConnectionsManagerService extends EventEmitter implements OnModuleI
...
+      }
+    )
+    this.socketService.on(
+      SocketActionTypes.DELETE_FILES_FROM_CHANNEL,
+      async (payload: DeleteFilesFromChannelSocketPayload) => {
+        this.logger(`socketService - ${SocketActionTypes.DELETE_FILES_FROM_CHANNEL}`, payload)
+        await this.storageService?.deleteFilesFromChannel(payload)
+        // await this.deleteFilesFromTemporaryDir() //crashes on mobile, will be fixes in next versions
+      }
+    )
```
Comment (leblowl) [inline packages/backend/src/nest/connections-manager/connections-manager.service.ts:530]: Also re-organizing

File: packages/backend/src/nest/connections-manager/connections-manager.service.ts [hunk 4]
```diff
@@ -511,11 +502,35 @@ export class ConnectionsManagerService extends EventEmitter implements OnModuleI
...
+    // TODO: Save community CA data in LevelDB. Perhaps save the
+    // entire Community type in LevelDB. We can probably do this once
+    // when creating the community.
+    this.socketService.on(
+      SocketActionTypes.SEND_COMMUNITY_CA_DATA,
+      async (payload: PermsData) => {
+        this.logger(`socketService - ${SocketActionTypes.SEND_COMMUNITY_CA_DATA}`)
+        this.registrationService.permsData = payload
+      }
+    )
```
Comment (leblowl) [inline packages/backend/src/nest/connections-manager/connections-manager.service.ts]: Here I renamed LAUNCH_REGISTRAR to SEND_COMMUNITY_CA_DATA and also the corresponding saga. Other than that it's just deleting things. It might be a good idea to split the rename into a separate PR, but I think this is fine too.

File: packages/backend/src/nest/registration/registration.functions.ts [hunk 5]
```diff
@@ -1,8 +1,8 @@
 import { createUserCert, keyFromCertificate } from '@quiet/identity'
 import { IsBase64, IsNotEmpty, validate } from 'class-validator'
-import { ErrorPayload, PermsData, SocketActionTypes, SuccessfullRegistrarionResponse } from '@quiet/types'
+import { ErrorPayload, PermsData, SocketActionTypes, type SuccessfulRegistrationResponse } from '@quiet/types'
```
Comment (leblowl) [inline packages/backend/src/nest/registration/registration.functions.ts:3]: Fix typo

**Review by EmiM — APPROVED** [review]
Nice, thanks for cleaning this up.
Could you look at those checks that did not pass?
